.monorepolint.config.mjs

```diff
@@ -17,9 +17,9 @@ const archetypeConfig = archetypes(
   (shared, rules) => {
     const baseScripts = {
       clean: "rimraf .turbo build dist lib test-output *.tgz tsconfig.tsbuildinfo",
-      lint: "eslint ./src ; dprint check --config $(find-up dprint.json) --allow-no-files",
+      lint: "eslint ./src && dprint check --config $(find-up dprint.json) --allow-no-files",
       "lint:fix":
```

Comment:
for this one it's kinda nice to be able to run these concurrently, but I guess we could refactor and turbo could manage that instead. 

Makes sense the fix can't be concurrent because the same files could be getting touched